apps-rendering/api-models/thrift/src/main/thrift/appsRendering.thrift

```diff
@@ -119,6 +119,19 @@ struct FootballContent {
     8: optional string venue
 }
 
+
+struct Newsletter {
+    1: required string identityName
```

Comment:
out of interest, how will `identityName` be used in Apps Rendering? Is it the same as `listId`?

Reply:
Ok, makes sense! I think with `thrift`, it's a good idea to keep the model as small as possible. We can always add fields later if we need them, but it's not possible to remove fields in this case as they're marked as required.